messages/cardano.proto

```diff
@@ -91,10 +98,16 @@ message CardanoSignTransactionRequest {
       repeated uint32 keypath = 1;
       bytes pool_keyhash = 2;
     }
+    message VoteDelegation {
+      repeated uint32 keypath = 1;
```

Comment:
in line with Address configuration, I'd consider explicitly stating that this is supposed to be `stake_keypath`

https://github.com/RostarMarek/bitbox02-firmware/pull/1/files#diff-7e05fd323f664298cafb095da6c3b84ddb139b8180f46259f517532d87b00ae4R40-R44

Reply:
validation is done here https://github.com/RostarMarek/bitbox02-firmware/blob/affe149c97ebfced46d12dbcf33941295981df15/src/rust/bitbox02-rust/src/hww/api/cardano/keypath.rs#L84 and then used in this here https://github.com/RostarMarek/bitbox02-firmware/blob/6cc02f149b0d284bfbfba017e38aff4c159d2f15/src/rust/bitbox02-rust/src/hww/api/cardano/sign_transaction/certificates.rs#L96